blog/docs/articles/about-serving-revision-timeout.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "about serving revision timeout"
+linkTitle: "about serving revision timeout"
+Author: "[Nader Ziada](https://twitter.com/nziada)"
+Author handle: https://github.com/nader-ziada
+date: 2022-07-20
+description: "Serving revision timeout different fields"
+type: "blog"
+---
+
```

Comment:
we don't use this frontmatter anymore - please see the other files in `articles` for reference

Reply:
fixed, thanks